src/lib/util.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Utility function which pluralizes a word given a number.
+ */
+export const pluralize = (word: string, num: number) => (num === 1 ? word : `${word}s`);
```

Comment:
nit: isn't there a better library for this? Something that might correctly pluralize `status` into `statuses`, and other exceptions?

Reply:
There are libraries for this, I'll take a look and see which would be a good fit. Since there's only one usage though, I might just take the function out and make the checkbox group's "All Label" be required so we don't add to the build size.

Reply:
Yeah, I'd lean in favor of taking the function out entirely.  We've made it _surprisingly_ far without pulling in a full-blown pluralization library, but for an OSS project, it seems good to avoid the implication that this will cover everything.

Reply:
I agree we should remove pluralize. @JonTheBurley My preference (if it works) would be to leave allLabel optional, but add the showAll prop and have it be dependent on whether allLabel is provided or not like the following:

```jsx
<CheckboxGroup
      key={name}
      onHandleChange={update}
      name={name}
      options={options}
      selected={value}
      allLabel={listFilterOptions.allLabel}
      showAll={Boolean(listFilterOptions.allLabel)}
  />
```

Reply:
Done. I have removed the pluralize function and removed the redundant showAll prop with just using the allLabel to show/hide the all checkbox based on its presence.